### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by huyluong22**
I have added logging for Controller and Service classes and removed all System.out.println statements. With this logging implementation, we can easily track the details of the application's workflow in production logs. The logs can be found in the logs folder, which includes the application.log file
image

### CODE REVIEW

File: userservice/src/main/resources/application.properties [hunk 1]
```diff
@@ -5,10 +5,6 @@ server.port=8080
 spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
 springfox.documentation.swagger.v2.path=/api/v2/api-docs
 
-# Redis configuration
-spring.data.redis.host=localhost
-spring.data.redis.port=6379
```
Comment (thaibinh156) [inline userservice/src/main/resources/application.properties:10]: Why did you remove the Redis configuration?

**Comment by huyluong22** [comment]
When I pulled the code to a new branch, the Redis configuration was not in my file. That’s why it was removed when I pushed my new code.
image
To verify, I created a test branch, pulled the code, and found that the Redis configuration was still missing 
image
Let me add the Redis configuration back

File: userservice/pom.xml [hunk 2]
```diff
@@ -102,23 +96,15 @@
 			<artifactId>hibernate-core</artifactId>
 			<version>6.2.2.Final</version>
 		</dependency>
-
-		<dependency>
-			<groupId>io.springfox</groupId>
-			<artifactId>springfox-swagger2</artifactId>
-			<version>3.0.0</version>
-		</dependency>
-
```
Comment (thaibinh156) [inline userservice/pom.xml:111]: Can you add the Swagger and Swagger UI dependency to this file?

Comment (huynhatluong22) [reply userservice/pom.xml:111]: sure

File: userservice/src/main/resources/application.properties (hunk 1 above)
Comment (SonNgocTruong) [reply userservice/src/main/resources/application.properties:10]: **FYI**, @huyluong22 , @thaibinh156 
Please keep the original value for configuration. If you want to customize your configuration to be compact with your local development machine, you can use environment variables set up on IntellJ.
For example:
image

**Review by SonNgocTruong — APPROVED** [review]
LGTM
